### ISSUE TITLE
Severe SARS-CoV-2 infection

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6201]
**Preferred term label:**
Severe SARS-CoV-2 infection

**Synonyms**
Severe COVID-19

**Definition (free text, please give PubMed ID)**
Unusually severe clinical course of SARS-CoV-2 infection, manifested clinically by features such as dyspnea and hypoxia with diffuse bilateral ground-glass opacities of the lungs on computed tomographic scan with progressive respiratory insufficiency necessitating oxygen supplementation or mechanical ventilation.
PMID:32706371

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0031691

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Immunodeficiency 74, COVID19-related, X-linked, OMIM:301051 

4/4 (PMID:32706371)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
